src/Host/Monitor/Impl/MainWindow.xaml

```diff
@@ -1,21 +1,21 @@
 ﻿<Window x:Class="Microsoft.R.Host.Monitor.MainWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:Microsoft.R.Host.Monitor"
-        mc:Ignorable="d"
-        Title="R Host Monitor" Height="350" Width="525">
-    <Grid>
+        Title="R Remote Services Monitor" Height="350" Width="560" ResizeMode="NoResize"
+        Background="#FFFCFCFC">
```

Comment:
If we explicitly apply light Aero somewhere (I don't see this in XAML), then we have accessibility issues with the app, because it doesn't respect high-contrast settings.

If we don't, then it's not a given that the theme is actually light. Windows has a "High Contrast Black" theme out of the box, that can be enabled in settings (Ease of Access | High Contrast).